packages/backend/src/routes/shoppingLists.js

```diff
@@ -22,14 +22,17 @@ import { joiValidator } from "../middleware/joiValidator.js";
 
 // Util
 import { wrapRequestWithErrorHandler } from "../utils/wrapRequestWithErrorHandler.js";
-import { NotFound } from "../utils/errors.js";
+import { NotFound, PreconditionFailed } from "../utils/errors.js";
 
 router.post(
   "/",
   cors(),
   MiddlewareService.validateSession(["user"]),
   MiddlewareService.validateUser,
   wrapRequestWithErrorHandler(async (req, res) => {
+    if (!req.body.title || req.body.title.length === 0) {
```

Comment:
noticed a test failed with. a 500 if no title is supplied, so added this wrapper like in recipes